### DISCUSSION TITLE
2 million mt cap in the EBS

### DISCUSSION THREAD

**Discussion by JaneSullivan-NOAA** [afsc-assessments/ssmesa-weekly#6] [General]
What's the history on this?

**Comment by zandjyo** [comment]
I think this paper kind of addresses the history/origin of the 2 million mt cap... as best as it has been documented anyway. [Fishery Science and Management  Objectives and Limitations - 1989 - Wooster - Biological Basis for Management of Commercial.pdf](https://github.com/afsc-assessments/ssmesa-weekly/files/11851150/Fishery.Science.and.Management.Objectives.and.Limitations.-.1989.-.Wooster.-.Biological.Basis.for.Management.of.Commercial.pdf)


From this paper:  "Socioeconomic conditions. Apparently when the sum of
the TAC's determined for biological reasons falls outside the OY range of 1.4
million to 2.0 million tonnes mentioned above, then the TAC's are adjusted for
"socioeconomic reasons" to force them back into that range. Again,
documentation is very confusing and hard to follow in any logical way."

**Comment by JaneSullivan-NOAA** [comment]
Thank you for sharing @zandjyo!
